Makefile

```diff
@@ -9,6 +9,10 @@ LDFLAGS = -ldflags "-X main.BuildTime=$(BUILD_TIME) -X main.GitCommit=$(GIT_COMM
 .PHONY: all
 all: audit test build
 
+.PHONY: fmt
+fmt:
+	go fmt ./...
```

Comment:
(suggestion): Not sure if we need this in the makefile. It will be handled by the linter